aimemalaika/Andela_Cycle_12_Chalenge_Two
# Apply feedback for adding the maintainability badge

## What does this PR do?
This PR is raised to add maintainability badge and consistency

## Description of the task to be completed
- add maintainability badge
## How should it be tested
- go to the repository https://github.com/aime19/Andela_Cycle_12_Chalenge_Two.git
check the badge in the read me
## Any background context to provide ?
 - N/A

## What are the relevant pivotal tracker stories
N/A
